webapp-django/crashstats/settings/base.py

```diff
@@ -715,6 +718,16 @@ def path(*dirs):
             'virtual_host': config('RABBITMQ_VIRTUAL_HOST', '/'),
             'port': config('RABBITMQ_PORT', 5672),
         },
+        'boto': {
+            'access_key': config('resource.boto.access_key', ''),
+            'bucket_name': config(
+                'resource.boto.bucket_name', 'crashstats'),
+            'prefix': config('resource.boto.prefix', ''),
+            'keybuilder_class': config(
+                'resource.boto.keybuilder_class',
+                'socorro.external.boto.connection_context.DatePrefixKeyBuilder'
+            ),
+        }
```

Comment:
This is to take advantage of what's in consul under the `socorro/common` prefix. 
But now that you mention it, why didn't we do the same with the key for rabbitmq for example?!

My mind is a bit frayed with this and I'll like just deploy and test this first and foremost. Perhaps iterate thereafter.